### ISSUE TITLE
Can you support read config  from spring boot application.properties?

### ISSUE THREAD

**Issue by Dreampie** [debop/hibernate-redis#64]
like this(application.yml):
```
spring:
  datasource:
  # ...
  jpa:
    showSql: true
    hibernate:
      ddlAuto: validate
    properties:
      javax:
        persistence:
          sharedCache:
            mode: ALL
      hibernate:
        cache:
          use_second_level_cache: true
          use_query_cache: true
          transaction:
            factory_class: org.hibernate.engine.transaction.internal.jdbc.JdbcTransactionFactory
          region:
            factory_class: org.hibernate.cache.redis.SingletonRedisRegionFactory
          redis:
            expiry_in_seconds: 0 #秒
```

**Comment by debop** [comment]
OK, I will make it.

**Comment by Dreampie** [comment]
thanks，and It is best to read the redis from spring boot config or use spring `RedisCacheManager`

**Comment by Dreampie** [comment]
yes，hibernate config is ok why not use spring redis config？

**Comment by Dreampie** [comment]
and in cloud config application.yml , this will work fine?

**Comment by GeBeater** [comment]
@Dreampie With the current pull request, I made it possible to configure hibernate via the "default" properties mechanism instead of the usage of an additional hibernate.properties file. In my use case, I am working with spring property files instead of yaml configuration which working fine. I guess that it should also work with yaml files because [Using YAML instead of Properties](http://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-external-config.html#boot-features-external-config-yaml) is a feature of spring. However I have not try it and also did not investigate the spring sources regarding this.

Regarding the redis configuration. In my humble opinion it is good to keep the library framework independently. So I decided to reuse the injected properties instead of fetching any settings from the RedisCacheManager because this required an additional spring dependency. However @debop , as project owner, can give us a clear statement.

The next step I am going to reach is, that it no more required to provide a redisson.yaml file. I try to find a solution that the settings contained in the redisson.yaml file can also used from files, maintained by spring. However this issue was not part of the pull request #67 .
